tools/python/boutiques/descriptor2func.py

```diff
@@ -0,0 +1,27 @@
+from boutiques.util.utils import loadJson
+from boutiques import validate, execute, prettyprint
+import json
+
+
+def function(descriptor, name=None, mode='launch'):
+
+    validate(descriptor)
+    descriptor_json = loadJson(descriptor)
+
+    def f(*args, **kwargs):
```

Comment:
Can we document that `args` is used for `bosh exec launch` runtime arguments, and invocations are passed in via `kwargs`? Just to make sure this is obvious to people.

Reply:
Maybe this would be done by creating the `__doc__` with details about this function wrapper, and appending the `prettyprint` to it for actual tool docs

Reply:
done